### ISSUE TITLE
Mouseover on pod logs hangs after click-through in pods list

### ISSUE THREAD

**Issue by puja108** [kubernetes/dashboard#877]
#### Issue details

The mousover details (grey cricle and text) stay on the logs button in the pod list once I click through to logs. They only go away after clicking through to another pod's logs.
##### Environment

rc+pod in Kube-Solo local env

```
Dashboard version: 1.1.0beta3
Kubernetes version: 1.2.4
Operating system: CoreOS 899
Node.js version:
Go version:
```
##### Steps to reproduce

This works in both `#/workload` (pods list) and `#/pod`
1. click on a pod's logs button
2. close the opening log window
3. mouseover the logs button again to see it's text
4. mouseover other logs buttons
##### Observed result

After step 2 the grey circle around the logs button stays.
After step 3 the text for the logs button also stays.
##### Expected result

The circle and text should vanish (mouseovers) again when the mouse is not over them.
##### Comments

The "hanging mouseover" moves to another logs button once that gets clicked.

These screenshots were taken after clicking the logs button of the kube-dns pod.
After step 2:
<img width="1270" alt="bildschirmfoto 2016-06-10 um 18 46 52" src="https://cloud.githubusercontent.com/assets/5468872/15972236/8c555f06-2f3d-11e6-8845-aafb381c2b50.png">

After step 3:
<img width="1274" alt="bildschirmfoto 2016-06-10 um 18 47 13" src="https://cloud.githubusercontent.com/assets/5468872/15972270/c000de84-2f3d-11e6-8dd5-033c933a3eab.png">
<img width="1271" alt="bildschirmfoto 2016-06-10 um 18 47 24" src="https://cloud.githubusercontent.com/assets/5468872/15972276/c7cdacb4-2f3d-11e6-8f00-d83d5705c3ed.png">

**Comment by puja108** [comment]
Clicking on an empty space on the page also makes the "hanging mouseovers" vanish again.

**Comment by bryk** [comment]
Oh,  looks annoying. Let's see what we can do with this.

**Comment by floreks** [comment]
Ok, this looks like angular material bug when used with `target="_blank", we should create an issue there. I've created example codepen to show the issue.

http://codepen.io/pen/?editors=1010

**Comment by bryk** [comment]
Awesome, let's see how angular material team responds to this

**Comment by maciaszczykm** [comment]
#1734 is a duplicate of it, we'll track it from there.
